### PR TITLE
Revert "Add missing features for WebGL2RenderingContext API"

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
Reverts mdn/browser-compat-data#8183.

### CODE REVIEW

**Review by wieseaka — CHANGES_REQUESTED** [review]
[]()

File: api/WebGL2RenderingContext.json [hunk 1]
```diff
@@ -431,53 +431,6 @@
           }
         }
       },
-      "bufferData": {
-        "__compat": {
-          "support": {
-            "chrome": {
```
Comment (wieseaka) [inline api/WebGL2RenderingContext.json:437]: Ja

Comment (wieseaka) [reply api/WebGL2RenderingContext.json:437]: ****

**Review by wieseaka — APPROVED** [review]

**Review by Elchi3 — APPROVED** [review]
Thanks!

**Pull request merged**